include/nlohmann/json.hpp

```diff
@@ -1922,6 +2101,18 @@ class basic_json
     // object inspection //
     ///////////////////////
 
+    /*!
```

Comment:
This should not stay like this. If we do want to control the behavior of `dump`, we need to add another parameter.

Reply:
This variable really needs to go. We can control `dump` with a parameter, and `operator<<` was never designed to have the same amount of control as `dump`.